bool are_at_plant
int32 plant_id
int32 location